### PR TITLE
nwdnww

### PR DESCRIPTION
**Pull request by Rogal255**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @Rogal255

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Rogal255 for on-time delivery

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,40 @@
...
+    int higher {0};
+
+    if(lhs > rhs)
+    {
+        higher = lhs >= 0 ? lhs : -lhs;
+    }
+    else
+    {
+        higher = rhs >= 0 ? rhs : -rhs;
+    }
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:13]: Use std::swap instead create new variable

if (lhs > rhs) {
   std::swap(lhs, rhs)
}

File: homework/nwd-nnw/nwdNww.hpp [hunk 2]
```diff
@@ -1,11 +1,40 @@
...
-    return -1;
+    if(lhs == 0 || rhs == 0)
+    {
+        return 0;
+    }
+
+    int result {lhs};
+    while(true)
+    {
+        if(result%rhs == 0)
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:34]: Add space between operations like '+' '-' '!=' '==' etc...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**